examples/client-mithril-stake-distribution/Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "client-mithril-stake-distribution"
 description = "Mithril client stake distribution example"
-version = "0.1.0"
-authors = ["dev@iohk.io"]
+version = "0.1.1"
```

Comment:
Does this crate version need to be incremented? Even if the example code has not been modified.